org.blackcoinmore.blackmore-qt.json

```diff
@@ -0,0 +1,75 @@
+    "id": "org.blackcoinmore.blackmore-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "command": "blackmore-qt",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "blackcoin-more",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/blackmored",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-cli",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-tx"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/CoinBlack/blackcoin-more/releases/download/v13.2.1/blackcoin-more-13.2.1-x86_64-linux-gnu.tar.gz",
+                    "sha256": "3681da3217019e202b887ab5d34f2a5ad3a1dd98368d3c7362f72753c0722184"
+                },
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "url": "https://github.com/CoinBlack/blackcoin-more/releases/download/v13.2.1/blackcoin-more-13.2.1-aarch64-linux-gnu.tar.gz",
+                    "sha256": "bd2a5f0cacfe170cdae814f64e41f1904a08911da4b09629b2441d74166dc780"
+                }
+            ]
+        },
+        {
+            "name": "misc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "desktop-file-edit --set-key=\"Icon\" --set-value=${FLATPAK_ID} blackmore-qt.desktop",
+                "install -Dp -m 644 blackmore-qt.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dp -m 644 blackcoin.svg /app/share/icons/hicolor/scalable/apps/org.blackcoinmore.blackmore-qt.svg",
+                "install -Dp -m 644 org.blackcoinmore.blackmore-qt.metainfo.xml /app/share/metainfo/org.blackcoinmore.blackmore-qt.metainfo.xml",
+                "bash install_wrappers.sh"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/coinblack/blackmore-flatpak/main/blackmore-qt.desktop",
+                    "sha256": "21d6c3498f9b2f34907751a0d6e5a3475d38c843afae24263c930c2f37db551c"
+                },
+                {
+                    "type": "file",
+                    "path": "org.blackcoinmore.blackmore-qt.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/coinblack/blackmore-flatpak/main/blackcoin.svg",
```

Comment:
you should pin this to the tag.

Reply:
you mean, add the files into this repo?

Reply:
no. That's not what I mean. I mean you should get the revision specific for the tag. This currently get whatever is in the main branch, the name time it changes that break the build.

Reply:
this repo is just to host the flies for the flatpak. there is no tag, just main branch

Reply:
then the current commit.

It should really be upstream ie in the main repo anyway.

Reply:
neither .desktop, icon or appstream file are specific to flathub/flatpak.
The former two are part of 20+ old standard that any desktop app should have.

Reply:
yes, I understand what you mean. I will add them to the contrib folder under the Blackcoin More repo. Makes sense!